docs/Core.rst

```diff
@@ -319,6 +336,11 @@ on UNIX-like systems:
 
   DFHACK_SOME_VAR=1 ./dfhack
 
+- ``DFHACK_DISABLE``: if set, DFHack will not initialize, not even to redirect
+  :file:`stdout.txt` or :file:`stderr.txt`. This is provided as an alternative
```

Comment:
Nitpick:
```suggestion
  standard output or standard error. This is provided as an alternative
```
DF (or is it SDL?) only writes to the .txt files on Windows. (Admittedly my information is based on 0.47, so maybe it will change once those ports come out for v50)